openedx/core/djangoapps/oauth_dispatch/management/commands/edx_clear_expired_tokens.py

```diff
@@ -90,6 +90,12 @@ def handle(self, *args, **options):
         refresh_expire_at = self.get_expiration_time(now)
 
         if options['refresh-tokens']:
+            # remove revoked RefreshTokens
+            revoked = RefreshToken.objects.filter(revoked__lt=refresh_expire_at).exclude(
```

Comment:
I would gate the revoked refresh token deletion on a different flag also, so we can parallelize that option or generally experiment a good amount with different combinations of toggles on-off. This motivated by our still being billions of rows deep in this issue and now sure which tactic is going to get us back to a good steady-state